### PR TITLE
Add ETH Riyadh 2024 event information

### PR DESCRIPTION
**Pull request by chenjiali-april**
This PR wants to add the ETH Riyadh 2024 event details to the ETH Events website. The event will take place in Riyadh, Saudi Arabia, from December 1st to December 3rd, 2024. It will focus on uniting Middle Eastern developers and blockchain enthusiasts to collaborate and innovate within the Ethereum ecosystem.

### CODE REVIEW

**Comment by chenjiali-april** [comment]
Hi there! I hope you’re doing well. I wanted to kindly remind you about this PR, which adds the details of the ETH Riyadh 2024 event to the ETH Events website. The event is scheduled for December 1st to 3rd in Riyadh, Saudi Arabia, and will bring together Middle Eastern developers and blockchain enthusiasts.

Please let me know if there’s anything I can do to help move this forward or if any additional information is needed. Your review and feedback would be greatly appreciated!

Thank you for your time and support!

File: src/data/community-events.json [hunk 1]
```diff
@@ -647,6 +647,15 @@
...
-    "imageUrl": "https://ethvenice.com/images/banner.jpg"
+    "imageUrl": "https://www.ethvenice.com/images/logo.png"
+  },
+  {
+    "title": "ETH Riyadh 2024",
+    "href": "https://ethriyadh.com/",
+    "location": "Riyadh, Saudi Arabia",
+    "description": "ETH Riyadh 2024 unites Middle Eastern developers and blockchain enthusiasts to explore and advance Ethereum technology. Join us for an exciting event that fosters collaboration and innovation in the blockchain ecosystem.",
+    "startDate": "12/1/2024",
+    "endDate": "12/3/2024",
```
Comment (minimalsm) [inline src/data/community-events.json:658]: dates must follow this convention 

```suggestion
    "startDate": "2024-12-1",
    "endDate": "2024-12-3",
```

**Review by minimalsm — APPROVED** [review]
LGTM, thanks @chenjiali-april

File: src/data/community-events.json [hunk 2]
```diff
@@ -647,6 +647,15 @@
     "description": "Join the second edition of the event in the center of Venice, to learn more about Blockchain, Technology and with an eye towards art.",
     "startDate": "2024-12-02",
     "endDate": "2024-12-03",
-    "imageUrl": "https://ethvenice.com/images/banner.jpg"
+    "imageUrl": "https://www.ethvenice.com/images/logo.png"
```
Comment (minimalsm) [inline src/data/community-events.json:650]:
```suggestion
    "imageUrl": "https://ethvenice.com/images/banner.jpg"
```

**Comment by minimalsm** [comment]
@all-contributors please add @chenjiali-april for events, docs

**Comment by chenjiali-april** [comment]
> @all-contributors please add @chenjiali-april for events, docs

THANKS! I have joined the discord! Also, the PR has been merged, but the webpage hasn’t updated yet. Could you let me know when the changes will go live?

**Comment by chenjiali-april** [comment]
> LGTM, thanks @chenjiali-april

Hi minimalsm, I wanted to kindly follow up on my previous question regarding the production environment. I noticed the PR has been merged into the dev environment. Could you please let me know when the changes will be deployed to production?  So that we can see our event on the Ethereum Event website, thanks again for your help!

**Pull request merged**